src/components/StoreInfo/StoreMenuTab.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { BrowsePageSeller } from '../../utilities';
+
+export interface Props {
+  getStoreImages: Function;
+  seller: BrowsePageSeller;
+}
+
+const StoreMenu: React.SFC<Props> = ({ seller, getStoreImages }) => {
+  const images = getStoreImages(seller.seller_id);
+
+  return (
+    <section>{<img src={images.menu} alt="store-menu" width="100%" />}</section>
```

Comment:
Should the width here be moved to a style?

Reply:
ya i'll move them